UI/obs-frontend-api/obs-frontend-api.h

```diff
@@ -62,6 +62,8 @@ enum obs_frontend_event {
 	OBS_FRONTEND_EVENT_SCENE_COLLECTION_RENAMED,
 	OBS_FRONTEND_EVENT_THEME_CHANGED,
 	OBS_FRONTEND_EVENT_SCREENSHOT_TAKEN,
+
+	OBS_FRONTEND_EVENT_STREAMING_PREPARING,
```

Comment:
With how messy service and streaming output codebase is becoming, I would be heavily against adding any Frontend API unless it's for untangling the whole broadcast code and the whole "golive" stuff and put most of it in the service code.

Reply:
since there's currently nothing using this event it should be ok to just remove it; let me know if you want me to do that

Reply:
Removed in a fixup commit after https://github.com/obsproject/obs-studio/pull/10633#event-12771825219

Reply:
Resolving.